docs/advanced-features/compiler.md

```diff
@@ -187,6 +187,37 @@ First, update to the latest version of Next.js: `npm install next@latest`. Then,
 }
 ```
 
+### Emotion
+
+We're working to port `@emotion/babel-plugin` to the Next.js Compiler.
+
+First, update to the latest version of Next.js: `npm install next@latest`. Then, update your `next.config.js` file:
+
+```js
+// next.config.js
+
+module.exports = {
+  compiler: {
+    //
+    emotion: boolean | {
```

Comment:
Is this good to go straight to stable, or should it be tested out as experimental first?

Reply:
Oh, I'll move it into Experimental Features section